src/AppInstallerCommonCore/Filesystem.cpp

```diff
@@ -62,6 +62,14 @@ namespace AppInstaller::Filesystem
         return (GetVolumeInformationFlags(path) & FILE_SUPPORTS_REPARSE_POINTS) != 0;
     }
 
+    bool PathEscapesBaseDirectory(const std::filesystem::path& path, const std::filesystem::path& base)

```

Comment:
Explicit unit tests for this function please.

Reply:
Added unit test for this function